specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -24574,6 +24960,41 @@
       "x-ms-client-name": "ServerlessEndpointProperties",
       "additionalProperties": false
     },
+    "ServerlessEndpointCapacityReservation": {
+      "required": [
+        "capacityReservationGroupId"
+      ],
+      "type": "object",
+      "properties": {
+        "capacityReservationGroupId": {
+          "description": "[Required] Specifies a capacity reservation group ID to allocate capacity from.",
+          "minLength": 1,
+          "pattern": "[a-zA-Z0-9_]",
+          "type": "string"
+        },
+        "endpointReservedCapacity": {
+          "format": "int32",
+          "description": "Specifies a capacity amount to reserve for this endpoint within the parent capacity reservation group.",
+          "type": "integer",
+          "x-nullable": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "ServerlessEndpointStatus": {
+      "type": "object",
+      "properties": {
+        "metrics": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
Please make this read only as was discussed in office hours.

Reply:
added readonly